Mapify/SceneInitializers/GameContent/VanillaAssetSetup.cs

```diff
@@ -12,21 +13,28 @@ public override void Run()
         {
             foreach (VanillaObject vanillaObject in Object.FindObjectsOfType<VanillaObject>())
             {
-                GameObject go = vanillaObject.Replace(keepChildren: vanillaObject.keepChildren, rotationOffset: vanillaObject.rotationOffset);
-
+                //only these belong in the gamecontent scene
                 switch (vanillaObject.asset)
                 {
+                    case VanillaAsset.CareerManager:
+                    case VanillaAsset.JobValidator:
+                    case VanillaAsset.TrashCan:
+                    case VanillaAsset.Dumpster:
+                    case VanillaAsset.LostAndFoundShed:
+                    case VanillaAsset.WarehouseMachine:
+                    case VanillaAsset.PlayerHouse:
+                    case VanillaAsset.PitStopStationCoal1:
+                    case VanillaAsset.PitStopStationCoal2:
+                    case VanillaAsset.PitStopStationWater1:
+                    case VanillaAsset.PitStopStationWater2:
                     case VanillaAsset.StationOffice1:
                     case VanillaAsset.StationOffice2:
                     case VanillaAsset.StationOffice3:
                     case VanillaAsset.StationOffice4:
                     case VanillaAsset.StationOffice5:
                     case VanillaAsset.StationOffice6:
                     case VanillaAsset.StationOffice7:
-                        // todo: make this show in the correct location instead of removing it. Then we can also get rid of this switch.
-                        Transform youAreHereFlag = go.transform.FindChildByName("PinRed");
-                        if (youAreHereFlag != null)
-                            Object.Destroy(youAreHereFlag.gameObject);
```

Comment:
Were these removed intentionally? Without it the pin will appear in the same spot for all stations.

Reply:
Yes, that was intentional, it doesn't seem to do anything. Even without these lines, no red pin is placed anywhere:

![afbeelding](https://github.com/Insprill/dv-mapify/assets/18124323/e2372576-be0e-4ab2-b892-f0e0e788ce2a)